lib/HuTrack/engine/HuTrack_parser.asm

```diff
@@ -238,6 +238,7 @@ HuTrackEngine.Parser:
 HuTrackEngine.ParseEntry:
 
         cly
+        stz HuTrack.channel.temp.fxArg1,x

```

Comment:
Here, I initialize the argument to 00. Just in case FX is set in the mask, but the FX arg value is not.